base/strings/util.jl

```diff
@@ -477,24 +477,31 @@ julia> hex2bytes(a)
 """
 function hex2bytes end
 
-hex2bytes(s::AbstractString) = hex2bytes(Vector{UInt8}(String(s)))
-hex2bytes(s::AbstractVector{UInt8}) = hex2bytes!(Vector{UInt8}(uninitialized, length(s) >> 1), s)
+hex2bytes(s::AbstractString) = hex2bytes(String(s))
+hex2bytes(s::Union{String,AbstractVector{UInt8}}) = hex2bytes!(Vector{UInt8}(uninitialized, length(s) >> 1), s)
+
+_nbytes(s::String) = sizeof(s)
+_nbytes(s::AbstractVector{UInt8}) = length(s)
```

Comment:
isn't this also `sizeof`? I thought we define that `sizeof` means `nbytes`

Reply:
I was being super paranoid, but you're probably right.